odl/contrib/fom/supervised.py

```diff
@@ -636,7 +634,88 @@ def psnr(data, ground_truth, normalize=False):
     else:
         return 20 * np.log10(max_true) - 10 * np.log10(mse_result)
 
+
+def haarpsi(data, ground_truth, a=4.0, c=None):
```

Comment:
Maybe we can go with the default `4.2` as in the article.

Reply:
We should indeed follow the article.